morf-oracle/src/main/java/org/alfasoftware/morf/jdbc/oracle/OracleDialect.java

```diff
@@ -1312,7 +1312,12 @@ protected String selectStatementPreFieldDirectives(SelectStatement selectStateme
   protected String updateStatementPreTableDirectives(UpdateStatement updateStatement) {
     for (Hint hint : updateStatement.getHints()) {
       if (hint instanceof UseParallelDml) {
-        return "/*+ ENABLE_PARALLEL_DML PARALLEL */ "; // only the single hint supported so return immediately
+        StringBuilder builder = new StringBuilder("/*+ ENABLE_PARALLEL_DML PARALLEL");
+        builder.append(((UseParallelDml) hint).getDegreeOfParallelism()
+                .map(d -> "(" + d + ")")
+                .orElse(""));
+        builder.append(" */ ");
+        return builder.toString(); // only the single hint supported so return immediately

```

Comment:
Note that this will have to be reworked as soon as someone adds a second hint to UpdateStatement.
Let's use this opportunity and code this in the same way it is done for InsertStatement.
I.e. Create a StringBuilder outside the if block, return outside the if block.
For the future generations... :)